allPaths.ts

```diff
@@ -7,7 +7,7 @@ const protectedPaths = [DASHBOARD, PROFILE, SETTINGS];
 const allPaths = {
   DASHBOARD,
   HOME: "/",
-  LOGIN: "/api/auth/signin",
```

Comment:
This was required to prevent NextJS from using its default login page, which I didn't declare anywhere; it comes with the framework. I added this `/login` path only after creating the component for it.